src/parse/value/css_function.rs

```diff
@@ -64,7 +64,12 @@ impl<'a> Parser<'a> {
 
     pub(super) fn parse_progid(&mut self) -> SassResult<String> {
         let mut string = String::new();
-        let mut span = self.toks.peek().unwrap().pos();
+        let mut span = match self.toks.peek() {
+            Some(token) => token.pos(),
+            None => {
+                return Err(("Unexpected end of tokenstream.", self.span_before).into());
```

Comment:
The `dart-sass` error message in this case is

```
Error: expected "(".
```

Could we change the message to that (exluding the `Error:` bit)?

Reply:
Wasn't sure if you wanted to also copy the error message exactly, now it's clear you want to have the exact same behavior as dart-sass, down to the error messages. Will keep this in my going forward!